front/src/app/components/forms/autocomplete/AppellationAutocomplete.tsx

```diff
@@ -24,10 +24,12 @@ const useAppellationAutocomplete = (
   const appellationLocatorSelector = useAppSelector(
     makeAppellationLocatorSelector(locator),
   );
-  const options = appellationLocatorSelector?.suggestions.map((suggestion) => ({
-    value: suggestion,
-    label: suggestion.appellation.appellationLabel,
-  }));
+  const options = appellationLocatorSelector?.suggestions
+    ? appellationLocatorSelector.suggestions.map((suggestion) => ({
+        value: suggestion,
+        label: suggestion.appellation.appellationLabel,
+      }))
+    : [];
```

Comment:
c'est peut-être plus lisible comme ça :

```suggestion
  const options = (appellationLocatorSelector?.suggestions ?? []).map((suggestion) => ({
        value: suggestion,
        label: suggestion.appellation.appellationLabel,
      }));
```

Reply:
Dans ce cas, à reporter sur tous les autocompletes @clement-duport